src/backend/joanie/debug/views.py

```diff
@@ -79,6 +82,81 @@ class DebugMailSuccessPaymentViewTxt(DebugMailSuccessPayment):
     template_name = "mail/text/order_validated.txt"
 
 
+class DebugMailSuccessInstallmentPaid(TemplateView):
+    """Debug View to check the layout of the success installment payment by email"""
+
+    def get_context_data(self, **kwargs):
+        """
+        Base method to prepare the document context to render in the email for the debug view.
+        Usage reminder :
+            /__debug__/mail/installment_paid_html
+        """
+        context = super().get_context_data()
+        product = ProductFactory(price=Decimal("1000.00"))
+        product.set_current_language("en-us")
+        product.title = "Test product"
+        product.set_current_language("fr-fr")
+        product.title = "Test produit"
+        product.save()
```

Comment:
Shorter way to do that
```suggestion
	        product = ProductFactory(price=Decimal("1000.00"), title="Test product")
        product.translations.create(language_code="fr-fr", title="Test produit")
```